playingaway.R

```diff
@@ -8,134 +8,157 @@ library(ggplot2)
 library(ggthemes)
 library(tidyr)
 
+# Get team data ----
+team_state_csv <-
+    "team,state
```

Comment:
States need to be in full, as this is how they are stored at afltables.com.au

Will having two state variables with the same name (here and in venues) create problems with merging?